### PR TITLE
Adding an SCC when enabling unsafe sysctls

### PR DESCRIPTION
**Pull request by lamek**
https://bugzilla.redhat.com/show_bug.cgi?id=1754403

### CODE REVIEW

**Comment by xiaojiey** [comment]
@lyman9966 Could you help to review? Thanks.

**Comment by lyman9966** [comment]
line 141 and 142 have format issue, should be: 
allowedUnsafeSysctls:
  - kernel.msg*
  - net.ipv4.route.min_pmtu

File: admin_guide/sysctls.adoc [hunk 1]
```diff
@@ -133,6 +133,25 @@ kubeletArguments:
     - "kernel.msg*,net.ipv4.route.min_pmtu"
 ----
 
+. Add the required sysctl parameter to the `allowedUnsafeSysctls` list in a security context constraint (SCC) object. It is recommened to use a new SCC object:
++
+[source,yaml]
+----
+[...]
+allowedUnsafeSysctls:
```
Comment (lyman9966) [inline admin_guide/sysctls.adoc:141]: line 141 and 142 have format issue, should be:
allowedUnsafeSysctls:
  - kernel.msg*
  - net.ipv4.route.min_pmtu

**Comment by lamek** [comment]
Thanks @xiaojiey.
@lyman9966 I made the formatting change you suggested. Do you mind taking a second look?
https://github.com/openshift/openshift-docs/pull/20914/files#diff-f720aaae3d7c19cb92367128f870c567R141

```
[...]
allowedUnsafeSysctls:
  - "kernel.msg*"
  - "net.ipv4.route.min_pmtu"
[...]
```

**Comment by lamek** [comment]
Closing this PR b/c we're retiring master-3. I'll follow up with changes in this PR to the enterprise-3.11 branch.
https://github.com/openshift/openshift-docs/pull/21135

**Pull request closed without merging**